src/Raven.Server/Documents/TimeSeries/TimeSeriesReader.cs

```diff
@@ -152,7 +152,10 @@ internal bool Init()
             if (_from > _to)
                 return false;
 
-            using (var holder = new TimeSeriesSliceHolder(_context, _documentId, _name).WithBaseline(_from))
+            var baseline = _from;
+            AddOffsetIfNeeded();
```

Comment:
can we move this below the holder to avoid the `baseline` var?

Reply:
done